_data/nav.yml

```diff
@@ -4,27 +4,27 @@
     - subtitle: "generator-cf"
       subhref: "https://github.com/cfpb/generator-cf"
     - subtitle: "Compiled source"
-      subhref: "https://cfpb.github.io/capital-framework/releases/capital-framework-latest.zip"
+      subhref: "http://registry.npmjs.org/capital-framework/-/capital-framework-3.1.1.tgz"
```

Comment:
Yeah, Travis [builds CF](https://github.com/cfpb/capital-framework/blob/canary/scripts/npm/prepublish/lib/build.js#L8) before publishing it to npm (so that the published archives contain `dist/` assets).

Linking to the v3.1.1 npm archive is a stopgap until we figure out GH releases.